src/us/kbase/workspace/test/kbase/IdentifierUtilsTest.java

```diff
@@ -777,20 +768,17 @@ public void sucessObjectSpecIncludedFalseStrict() throws Exception {
 				.withToObjRefPath(new LinkedList<String>()));
 		final List<ObjectIdentifier> ret = IdentifierUtils .processObjectSpecifications(oss);
 		assertThat("incorrect return size", ret.size(), is(1));
-		final ObjIDWithRefPathAndSubset oi = (ObjIDWithRefPathAndSubset) ret.get(0);
+		final ObjectIdentifier oi =  ret.get(0);
 		
 		checkObjectIdentifier(oi, "foo", null, "bar", null, null, "foo/bar", "bar", false);
-		assertThat("incorrect hasChain()", oi.hasRefPath(), is(false));
+		assertThat("incorrect hasRefpath()", oi.hasRefPath(), is(false));
```

Comment:
If you are feeling particularly pedantic/detail-oriented, `incorrect hasRefpath()` has the wrong casing.

Reply:
Fixed locally, it'll show up about 5 PRs from now